app/templates/package.json

```diff
@@ -2,19 +2,25 @@
   "name": "<%= moduleName %>",
   "version": "<%= moduleVersion %>",
   "description": "<%= moduleDesc %>",
-  "main": "index.es5.js",
+  "main": "dist/index.js",
+  "files": [
+    "bin/",
+    "dist/",
+    "LICENSE"
+  ],
   "scripts": {
-    "coverage": "isparta cover _mocha index.js -- --require babel/register",
-    "precoveralls": "npm run coverage",
-    "coveralls": "coveralls < coverage/lcov.info",
-    "test": "mocha --require babel/register",
+    "clean": "rimraf dist",
+    "lint": "eslint src test",
+    "build": "npm run clean && npm run lint && babel --optional runtime src -d dist",
+    "test": "npm run build && babel-node test/*.js",
```

Comment:
It's a plumbing script, but after working on these for a few days and also today I came up with an improved workflow which I published in my generator. I think we can close this PR. But I am pretty sure you can benefit from my research on using tape with coveralls and ES6. It seems simple now, but that was a good chunk of my weekend :laughing: